### PR TITLE
test/boost/cql_query_test: enable `parallelized_aggregation`

### PR DESCRIPTION
**Pull request by Jadw1**
Run tests for parallelized aggregation with `enable_parallelized_aggregation` set always to true, so the tests work even if the default value of the option is false.

### CODE REVIEW

File: test/boost/cql_query_test.cc [hunk 1]
```diff
@@ -5218,8 +5218,16 @@ SEASTAR_TEST_CASE(timeuuid_fcts_prepared_re_evaluation) {
     });
 }
 
+template<typename Func>
+static future<> with_parallelized_aggregation_enabled(Func&& func) {
```
Comment (avikivity) [inline test/boost/cql_query_test.cc:5222]: Please use std::function to reduce template proliferation.

Comment (avikivity) [reply test/boost/cql_query_test.cc:5222]: and add _thread to the name, to reduce the chance of confusion.

Comment (Jadw1) [reply test/boost/cql_query_test.cc]: changed

Comment (avikivity) [reply test/boost/cql_query_test.cc]: > and add _thread to the name, to reduce the chance of confusion.

You forgot this

Comment (Jadw1) [reply test/boost/cql_query_test.cc]: The page didn't refresh, updated

**Review by avikivity — APPROVED** [review]

**Review by denesb — APPROVED** [review]

**Pull request closed without merging**